docs/src/pages/[platform]/components/alert/react.mdx

```diff
@@ -131,7 +132,7 @@ Use the `onDismiss` prop to pass a function that will run when the Alert is dism
   </ExampleCode>
 </Example>
 
-## CSS Styling
+## Styling
```

Comment:
Updating this title to be inline with other component pages (because this section is more than just about CSS)